src/sympc/protocol/spdz/spdz.py

```diff
@@ -50,18 +61,33 @@ def mul_master(x, y, op_str):
     return shares
 
 
-""" Functions that are executed at a party """
+""" Functions that are executed at each party that holds shares """
+
 
+def mul_parties(
```

Comment:
Maybe we should isolate the code running on shares and the one running on ShareCC? Maybe again this is thoughts for later.